Simplenote/NoteEditorViewController.m

```diff
@@ -599,4 +581,50 @@ - (void)insertChecklistAction:(id)sender
     }
 }
 
+#pragma mark - New Note
+
+- (void)duplicateNote:(nullable Note *)sender
```

Comment:
duplicateNote is called with the selected note on NoteViewController.swift:856
```
@IBAction
    func duplicateNote(_ sender: Any) {
        guard let note = selectedNotes.first else {
            return
        }

        noteEditorViewController.duplicateNote(note)
        // TODO: Set correct analytic event
    }
```
We could remove the parameter and always use the `self.note` value of NoteEditorViewController. It's that ok?

Reply:
- That sounds super, we could just drop the parameter 😄 
- Perhaps `validateListDuplicateNoteMenuItem` should also check for `selectedNotes.count == 1`. Allowing for Duplicate, when there's more than one entry selected (and only duplicating the first one) would probably not be what I'd expect. WDYT?
- Do you think we could rename the `@IBAction func duplicateNote` to `duplicateNoteWasPressed` ? (so that it looks exactly the same as the other IBAction handlers).

Also, thank you sir!

Reply:
Done.
I'm changed the methods names and also added the `selectedNotes.count == 1` condition, not only on `validateListDuplicateNoteMenuItem` but also on `validateSystemDuplicateNoteMenuItem` to keep a consistence on the logic.

BTW: I renamed `validateListDuplicateNoteMenuItem` to `validateEditDuplicateNoteMenuItem` to match the existing method names